src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -807,4 +811,71 @@ export class PatientViewPageStore {
         this.ajaxErrors = [];
     }
 
+    readonly trialMatches = remoteData<ITrialMatch[]>({
+        invoke: () => {
+            return fetchTrialMatchesUsingPOST({mrn: this.patientId});
+        },
+        onError: (err: Error) => {
```

Comment:
lets remote these onError handlers.  We probably don't want it to fail silently

Reply:
Remove all onError handlers.